readme.md

```diff
@@ -80,40 +95,46 @@ Likewise when we write queries, we use **Verbs** to describe the specific data
 we want.
 
 Essentially, in order to store and retrieve information, a lot of what we do
-today will look like some form of the equation:
+today in ruby will look like some form of the equation:
 
  > **Noun** + **Verb** = **Data**
+ > Class.method # Class (Noun) method (Verb)
+ > Taco.all # returned all Taco data! 🌮 🌮 🌮 🌮 🌮 🌮 🌮 🌮 🌮 🌮 🌮 🌮
 
 With the help of Active Record, we can begin to write programs that follow this
 simple pattern to manipulate data.
 
-### Convention Over configuration (ST-WG - 10 / 35)
+### Convention Over configuration (10 / 35)
 
-Before we get started with code, I want to highlight a reoccurring theme with Active Record and Rails in general. You'll often here us say Convention over Configuration.
+Before we get started with code, I want to highlight a reoccurring theme with Active Record and Rails in general. You'll often hear us say, "Convention Over Configuration." Before we discuss the concept as a class, take 30 seconds to think about what that phrase means--why *might* we prefer convention over configuration?
 
 <details>
 <summary>**Question:**  Without getting into the specifics of AR, what do you think we mean by convention over configuration?</summary>
 <br>
 
 ```
-Basically Active Record and Rails, and other frameworks have a whole bunch of conventions that they follow so that you do not have to mess with different configuration details later. These conventions exist because developers agree on best practices, and therefore allows us to spend less time trying to configure, when there all ready is an accepted way to do things. Some of the common ones we will encounter are naming conventions such as: plural vs single, capital or lower, camel or snake.
+ActiveRecord and Rails, and other frameworks have a whole bunch of conventions that they follow so that you do not have to mess with different configuration details later. These conventions exist because developers arrive at a degree of consensus on best practices. These road-tested conventions therefore allow us to spend less time trying to configure, when there already is an accepted way to do things. We inherit from the programmers who have come before us a well-designed, default configuration that spares us from many headaches that we'd encounter if we were building things from scratch (yikes!). Some of the common ones we will encounter are naming conventions such as: plural vs single, Capitalized/ALL_CAPS_SNAKE_CASE/lowercase, camelCase/kabob-case/snake_case. Obeying the naming conventions in ActiveRecord, particularly regarding what is singular vs. what is plural, saves you a good deal of headaches.
```

Comment:
- ~~degree of~~
- These road-tested conventions ~~therefore~~...
- No need for comma after "trying to configure"
- Would start a new paragraph when you get to "We inherit..."
- Missing a word somewhere here: "We inherit from the programmers who have come before us [???] a well-designed..."
- ~~particularly regarding what is singular vs. what is plural~~ (you already mentioned that convention)